multimodal/src/autogluon/multimodal/optimization/lit_pretrainer.py

```diff
@@ -0,0 +1,476 @@
+import copy
```

Comment:
Is it possible to rename the file as `lit_tabular_pretrainer.py`?

Reply:
Also, PR needs to be rebased, there are conflicts

Reply:
Yeah,`lit_pretrainer` is too general. Object detection also needs pretraining, but can't use this module. I think `lit_tabular_pretrainer ` may also be too general since this is mainly to pretrain ft-transformer.